cylc/flow/cfgspec/suite.py

```diff
@@ -383,6 +383,18 @@
             shut down once all tasks have passed this point. If this item
             is provided you can override it on the command line.
         ''')
+        Conf('stop after cycle point', VDR.V_CYCLE_POINT, desc='''
+            Set stop point. Shut down after all tasks have PASSED
+            this cycle point. Will be over-ridden by command line
+            ``--stop-cycle-point=POINT``
```

Comment:
Could you add a quick note onto the `--stop-cycle-point` option to say that it overrides the `stop after cycle point` configuration in the workflow.

There's no markdown/RST or referencing syntax in the CLI, yet, but once we've sorted out the CLI we can autodoc that too.